pallets/gear-program/src/lib.rs

```diff
@@ -286,14 +292,22 @@ pub mod pallet {
 
     #[pallet::storage]
     #[pallet::unbounded]
-    pub(crate) type MemoryPageStorage<T: Config> =
```

Comment:
renaming to MemoryPageStorage2 will cause question for future code readers. Where is MemoryPageStorage1? Why 2 ? Better to not rename

Reply:
Yep, agree, however it is the easiest way

Reply:
Why??)) what is the problem with **MemoryPageStorage** ?